pkg/controller/realmadmin/settings.go

```diff
@@ -109,13 +110,13 @@ func (c *Controller) HandleSettings() http.Handler {
 		}
 
 		// General
-		if v := form.Name; v != "" {
+		if v := strings.TrimSpace(form.Name); v != "" {
```

Comment:
beforeSave already does this 

https://github.com/google/exposure-notifications-verification-server/blob/main/pkg/database/realm.go#L197